rendering_resource_manager_service/config/management/rendering_resource_settings_manager.py

```diff
@@ -55,7 +55,8 @@ def create(cls, params):
                 modules=str(params['modules']),
                 process_rest_parameters_format=str(params['process_rest_parameters_format']),
                 scheduler_rest_parameters_format=str(params['scheduler_rest_parameters_format']),
-                graceful_exit=params['graceful_exit'])
+                graceful_exit=params['graceful_exit'],
+                wait_until_running=params['wait_until_running'])
```

Comment:
My question was why you're not converting graceful_exit and wait_until_running to booleans here as you do below.